Samples/All/BumbleRumble/Common/GameScreen.cpp

```diff
@@ -105,20 +106,31 @@ void GameScreen::HandleInput()
     auto inputManager = Managers::Get<InputManager>();
 
 #if defined(_XBOX_ONE) && defined(_TITLE)
-    if (inputManager->IsNewButtonPress(InputManager::GamepadButtons::View))
-    {
-        concurrency::create_task(
-            Windows::Xbox::UI::SystemUI::ShowVirtualKeyboardAsync(
-                L"This is a text message",
-                L"Send Text Message",
-                L"Enter text to send:",
-                Windows::Xbox::UI::VirtualKeyboardInputScope::Default
-            )
-        ).then([](Platform::String^ text)
-        {
-            Managers::Get<NetworkManager>()->SendTextAsVoice(BumbleRumble::WStrToStr(text->Data()));
-        });
-    }
+	if (inputManager->IsNewButtonPress(InputManager::GamepadButtons::View))

```

Comment:
GamepadButtons::View is being used twice here; the second condition will never hit.

Reply:
Good catch.